app/ts/library/utilities.ts

```diff
@@ -58,3 +56,11 @@ export function stringIncludes(source: string, search: string, caseSensitive?: b
 export function preventFocus(e: JSX.TargetedEvent<HTMLElement>) {
 	e.currentTarget.blur()
 }
+
+/**
+ * Match string equality
+ */
+export function areEqualStrings(a: string, b: string, caseSensitive?: true) {
```

Comment:
do you need non-case sensitive check? if so, you could just not call this function and use "a === b". And when you do, you could rename this function to "areEqualStringsCaseInsentive" or similar

Reply:
Tracking this suggestion to #213 to push needed functionality to main